jbpm-db-scripts/src/main/resources/db/ddl-scripts/db2/db2-jbpm-schema.sql

```diff
@@ -802,6 +804,7 @@
     create index IDX_NInstLog_pInstId on NodeInstanceLog(processInstanceId);
     create index IDX_NInstLog_nodeType on NodeInstanceLog(nodeType);
     create index IDX_NInstLog_pId on NodeInstanceLog(processId);
+	create index IDX_NInstLog_pId on NodeInstanceLog(workItemId);
```

Comment:
typo, it should be IDX_NInstLog_workItemId

Reply:
ok